multimedia/x265/Portfile

```diff
@@ -52,6 +52,7 @@ platform darwin 10 {
     # Rosetta build has to override CMAKE_SYSTEM_PROCESSOR: https://trac.macports.org/ticket/64528
     if {${build_arch} eq "ppc"} {
         configure.args-append -DOVERRIDE_SYSTEM_PROCESSOR="ppc"
+        configure.args-append -DENABLE_HDR10_PLUS=On
```

Comment:
Your change enables HDR10_PLUS when

- installing x265 on darwin 10 PowerPC, or
- when installing with the `+highdepth` variant on all platforms

That does not seem to make a lot of sense to me. Would you not only want to have it enabled when the `+highdepth` variant is enabled? If so, this line should be removed.

Reply:
I altered that. As I said in the ticket - this was originally done by someone else who had that arg in that location. I have now removed it and left it only in the location enabling +highdepth. I understand why the logic there is wrong.